v1/lib/baseOutput.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Max
+ * Date: 05.02.2016
+ * Time: 09:50
+ */
```

Comment:
Take a look at how we are documentating a file (take a look at the file I send to you via flowdock). You can set this as preset in your phpstorm. Ask me on how to do it.